File: JULIA/Cap06/c06ex04.jl
# c06ex04.jl

using Printf

print("Entre a quantidade de elementos da matriz: ")
n = parse(Int, readline())
println()

aluno = Array{String}(undef, n)

for i in 1:n
  print("Entre o ", @sprintf("%3d", i), "o. nome: ")
  aluno[i] = readline()
end

println("\nForam fornecidos os nomes:\n")

for i in 1:n
  println("Nome ", @sprintf("%3d", i), " = ", aluno[i])
end

print("\nTecle <Enter> para encerrar... ")
readline()

File: CSHARP/Cap06/c06ex04.cs
// c06ex04.cs

using System;
using System.Collections.Generic;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int N;
        List<string> A = new List<string>();

        Write("Entre a quantidade de elementos da matriz: ");
        N = int.Parse(ReadLine());
        WriteLine();

        for (int I = 0; I <= N - 1; ++I)
        {
            Write($"Entre o {I + 1}o. nome: ");
            A.Add(ReadLine());
        }

        WriteLine();
        WriteLine("Foram fornecidos os nomes:");
        WriteLine();

        for (int I = 0; I <= N - 1; ++I)
        {
            WriteLine($"Nome {I + 1} = {A[I]}");
        }

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
